Practical1/Calculator/Calculator/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Calculator
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int number1;
+            int number2;
+
+            Console.WriteLine("Enter first number: ");
+            number1 = Convert.ToInt32(Console.ReadLine());
```

Comment:
we can directly create int variables here no need to create first and then initialize